### PR TITLE
re #1963: Tracking bulk imported files

### PR DESCRIPTION
**Pull request by ivakegg**
* Added the generation of a JSON encode mappings.txt file in the import directory which contains a the mapping of files.

### CODE REVIEW

**Comment by milleruntime** [comment]
This is an enhancement and 1.10 is only going to have bug fix releases.

**Comment by ctubbsii** [comment]
I don't have enough knowledge of the bulk import code to understand exactly what this accomplishes, but it seems to me that this is a slight implementation change intended to leave files around to help troubleshoot, and does not have any associated API changes. Is that correct?

Although we're only doing bugfixes on 1.10, I don't necessarily have a problem including a change that helps troubleshooting for further bugfixes, if the change is small and low-risk. However, I did have a few questions about these proposed changes:
1. Does this risk leaving extra files scattered around the filesystem that aren't cleaned up or is this placed in a directory that is already subject to cleanup?
2. Why not use one of the JSON libraries (like Gson) on our class path instead of manually constructing a JSON blob? Is there a performance concern?
3. Does the new behavior of adding a new file risk breaking existing user code around bulk import?
4. How does this change impact the new bulk import code paths and behavior, if it does at all?

**Comment by ivakegg** [comment]
This is not any kind of api change but merely a mechanism by which the client can track where the bulk imported file went.  On some or our systems I installed a mechanism to track this information via the logs, so this is not really needed anymore.  I am closing this pull request.  The original ticket however still stands in which it would be good to facilitate a mechanism to track files through the system from bulk import through compactions and garbage collection.

**Comment by EdColeman** [comment]
Was the necessary info in the log files already, or did you add additional logging?

File: server/master/src/main/java/org/apache/accumulo/master/tableOps/BulkImport.java [hunk 1]
```diff
@@ -266,6 +277,26 @@ public Exception call() throws Exception {
         throw ex.get();
       }
     }
+
```
Comment (dlmarion) [inline server/master/src/main/java/org/apache/accumulo/master/tableOps/BulkImport.java:280]: Suggest using a serialization framework here to serialize an object to the file. Is GSON or JAXB on the classpath in 1.10?

Comment (ctubbsii) [reply server/master/src/main/java/org/apache/accumulo/master/tableOps/BulkImport.java:280]: I think they both are.

**Pull request closed without merging**